Mathlib/Tactic/Symm.lean

```diff
@@ -0,0 +1,73 @@
+import Lean
+
+namespace Mathlib.Tactic.Symm
+open Lean Meta
+
+initialize symmExtension : SimpleScopedEnvExtension (Name × Array DiscrTree.Key) (DiscrTree Name) ←
+  registerSimpleScopedEnvExtension {
+    name := `symm
+    addEntry := fun dt (n, ks) => dt.insertCore ks n
+    initial := {}
+  }
+
+def relationAppM?(expr: Expr) : MetaM (Option (Expr × Expr × Expr)) :=
```

Comment:
```suggestion
def relationAppM? (expr : Expr) : MetaM (Option (Expr × Expr × Expr)) :=
```
nit: style here and elsewhere

Reply:
Could you clarify what is the correct style?